src/main/java/it/gov/pagopa/payhub/activities/activity/ingestionflow/debtposition/SynchronizeIngestedDebtPositionActivityImpl.java

```diff
@@ -53,7 +65,17 @@ public String synchronizeIngestedDebtPosition(Long ingestionFlowFileId) {
                     PaymentEventType paymentEventType = null; //TODO task P4ADEV-2421
                     WorkflowCreatedDTO workflowCreatedDTO = workflowDebtPositionService.syncDebtPosition(debtPosition, false, paymentEventType);
 
-                    // TODO invoke workflow status (P4ADEV-2345)
+                    if(workflowCreatedDTO == null || workflowCreatedDTO.getWorkflowId() == null){
+                        errors.append("No synchronization workflow created for debt position with iupdOrg " + debtPosition.getIupdOrg() + "\n");
+                        return;
+                    }
+
+                    WorkflowExecutionStatus workflowExecutionStatus = workflowCompletionService.waitTerminationStatus(workflowCreatedDTO.getWorkflowId(), maxAttempts, retryDelayMs);
+
+                    if (!WORKFLOW_EXECUTION_STATUS_COMPLETED.equals(workflowExecutionStatus)) {
+                        errors.append("Synchronization workflow for debt position with iupdOrg " + debtPosition.getIupdOrg() + " terminated with error status.\n");
```

Comment:
it's better prepend the new line char
```suggestion
                        errors.append("\nSynchronization workflow for debt position with iupdOrg " + debtPosition.getIupdOrg() + " terminated with error status.");
```